### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by davinegrao** [Lofter1/anyflip-downloader#57] [Q&A]
Hey, sorry if this is inconvenient. I'm kind of a technology noob and I don't know anything about coding but I still need this. How do I get this running on Windows? I did the installation thing on PowerShell but where do I put the ''anyflip-downloader <url to book>'' command?

**Comment by MHarrig** [comment]
Sorry, no answer, but I am in the same boat.

**Comment by penalemesc** [comment]
You put it in the same place you downloaded it from
